src/ResourceManager/DataFactories/Commands.DataFactories/help/AzureRM.DataFactories.md

```diff
@@ -50,8 +53,11 @@ Encrypts sensitive data.
 ### [New-AzureRmDataFactoryGateway](New-AzureRmDataFactoryGateway.md)
 Creates a gateway for an Azure Data Factory.
 
+### [New-AzureRmDataFactoryGatewayAuthKey](New-AzureRmDataFactoryGatewayAuthKey.md)
+Creates auth key for an Azure Data Factory Gateway.
+
 ### [New-AzureRmDataFactoryGatewayKey](New-AzureRmDataFactoryGatewayKey.md)
-Creates a gateway key for an Azure Data Factory.
+Creates a gateway key for an Azure Data Factory. This cmdlet is deprecated, and you should use **New-AzureRmDataFactoryGatewayAuthKey** instead.
```

Comment:
@nicolaszhang if this cmdlet is being deprecated, can we add a warning to the first line of this cmdlet's execution letting users know that they should use `New-AzureRmDataFactoryGatewayAuthKey` instead?